quickstarts/microsoft.datashare/data-share-account/azuredeploy.parameters.us.json

```diff
@@ -2,11 +2,8 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```

Comment:
If you pull the location (and it works) you can pull this file since there are no differences